### ISSUE TITLE
Change warning token color in o2-new

### ISSUE THREAD

**Issue by yceballost** [Telefonica/mistica-design#2048]
## Description
Warning color is not accessible with white backgrounds. Although one color should never be the only way to solve a status problem, it is good that the base status color is accessible to reduce contrast problems as much as possible.

The only disadvantage is that we no longer use an official color from the o2 palette.

We will create new color in palete o2Orange60 = #F06C00

## Tasks

- [x] Update tokens
- [x] Update libraries
- [x] [Android PR](https://github.com/Telefonica/mistica-android/pull/406)
- [x] iOS PR
- [x] [Web PR](https://github.com/Telefonica/mistica-web/pull/1314)
